docs/contributing/Style-guide-for-Flutter-repo.md

```diff
@@ -1321,205 +1298,11 @@ any code that operates on all of them should operate on them in the
-      );
-
-  // ...
-}
-```
-
-
-### Prefer a maximum line length of 80 characters
-
-Aim for a maximum line length of roughly 80 characters, but prefer going over if breaking the
-line would make it less readable, or if it would make the line less consistent
-with other nearby lines. Prefer avoiding line breaks after assignment operators.
-
-```dart
-// BAD (breaks after assignment operator and still goes over 80 chars)
-final int a = 1;
-final int b = 2;
-final int c =
-    a.very.very.very.very.very.long.expression.that.returns.three.eventually().but.is.very.long();
-final int d = 4;
-final int e = 5;
-
-// BETTER (consistent lines, not much longer than the earlier example)
-final int a = 1;
-final int b = 2;
-final int c = a.very.very.very.very.very.long.expression.that.returns.three.eventually().but.is.very.long();
-final int d = 4;
-final int e = 5;
-```
-
-```dart
-// BAD (breaks after assignment operator)
-final List<FooBarBaz> _members =
-  <FooBarBaz>[const Quux(), const Qaax(), const Qeex()];
-
-// BETTER (only slightly goes over 80 chars)
-final List<FooBarBaz> _members = <FooBarBaz>[const Quux(), const Qaax(), const Qeex()];
-
-// BETTER STILL (fits in 80 chars)
-final List<FooBarBaz> _members = <FooBarBaz>[
-  const Quux(),
-  const Qaax(),
-  const Qeex(),
-];
-```
-
-
-### Indent multi-line argument and parameter lists by 2 characters
-
-When breaking an argument list into multiple lines, indent the
-arguments two characters from the previous line.
-
-Example:
-
-```dart
-Foo f = Foo(
-  bar: 1.0,
-  quux: 2.0,
-);
-```
-
-When breaking a parameter list into multiple lines, do the same.
-
-
-### If you have a newline after some opening punctuation, match it on the closing punctuation.
-
-And vice versa.
-
-Example:
-
-```dart
-// BAD:
-  foo(
-    bar, baz);
-  foo(
-    bar,
-    baz);
-  foo(bar,
-    baz
-  );
-
-// GOOD:
-  foo(bar, baz);
-  foo(
-    bar,
-    baz,
-  );
-```
-
-### Use a trailing comma for arguments, parameters, and list items, but only if they each have their own line.
-
-Example:
-```dart
-List<int> myList = [
-  1,
-  2,
-];
-myList = <int>[3, 4];
-
-foo1(
-  bar,
-  baz,
-);
-foo2(bar, baz);
-```
-
-If one of the items is a multi-line callback, collection literal,
-or switch expression, it can be added without a trailing comma.
-
-```dart
-// GOOD:
-foo(
-  bar,
-  baz,
-  switch (value) {
-    true  => ScrollDirection.forward,
-    false => ScrollDirection.reverse,
-    null  => ScrollDirection.idle,
-  },
-);
-
-// also GOOD:
-foo(bar, baz, switch (value) {
-  true  => ScrollDirection.forward,
-  false => ScrollDirection.reverse,
-  null  => ScrollDirection.idle,
-});
-
-// The same applies to collection literals and callbacks:
-foo(<String>[
-  'list item 1',
-  'list item 2',
-  'list item 3',
-]);
-
-Future.delayed(Durations.short1, () {
-  if (mounted && _shouldOpenDrawer) {
-    _drawerController.forward();
-  }
-});
-```
+### Prefer a maximum line length of 80 characters for comments and docs
```

Comment:
Is this still 80? Or 100?

Reply:
Currently, most of our doc comments are formatted to 80 chars. We could unify this to say they should also be 100 chars. I do want to avoid that people reflow existing comments to 100 chars, though, which would pollute PRs.

Reply:
I am going to update this to say to prefer to keep them under 100 chars with a note that existing docs limited to 80 chars should not be reflowed just because of this.

Reply:
SGTM!